pt/lessons/specifics/ecto.md

```diff
@@ -274,7 +264,7 @@ query =
 
 ### Fragmentos
 
-As vezes a API Query não é suficiente, por exemplo, quando precisamos de funções específicas para banco de dados. A função `fragment/1` existe para esta finalidade:
+Às vezes a API Query não é suficiente, por exemplo, quando precisamos de funções específicas para banco de dados. A função `fragment/1` existe para esta finalidade:
```

Comment:
Same here, I have doubts whether we use Query API or API Query.

Reply:
An alternative is to use "a API de Query", but I think that "Query API" fits better.